src/designer/designer_packages/elsa-workflows-designer/src/modules/flowchart/flowchart.tsx

```diff
@@ -1,27 +1,27 @@
 import 'reflect-metadata';
-import {Component, Element, Event, EventEmitter, h, Listen, Method, Prop, State, Watch} from '@stencil/core';
-import {first} from 'lodash';
-import {Edge, Graph, Model, Node, NodeView, Point} from '@antv/x6';
+import { Component, Element, Event, EventEmitter, h, Listen, Method, Prop, State, Watch } from '@stencil/core';
+import { first } from 'lodash';
+import { Edge, Graph, Model, Node, NodeView, Point } from '@antv/x6';
 import './shapes';
 import './ports';
-import {ActivityNodeShape} from './shapes';
-import {Activity, ActivityDeletedArgs, ActivityDescriptor, ActivitySelectedArgs, ChildActivitySelectedArgs, ContainerSelectedArgs, EditChildActivityArgs, GraphUpdatedArgs, WorkflowUpdatedArgs} from '../../models';
-import {createGraph} from './graph-factory';
-import {AddActivityArgs, Connection, Flowchart, FlowchartModel, FlowchartPathItem, LayoutDirection, RenameActivityArgs, UpdateActivityArgs} from './models';
-import {NodeFactory} from "./node-factory";
-import {Container} from "typedi";
-import {createActivityLookup, EventBus, flatten, PortProviderRegistry, walkActivities} from "../../services";
-import {ConnectionCreatedEventArgs, FlowchartEvents} from "./events";
-import {ContextMenuAnchorPoint, ContextMenuItem} from "../../components/shared/context-menu/models";
+import { ActivityNodeShape } from './shapes';
+import { Activity, ActivityDeletedArgs, ActivityDescriptor, ActivitySelectedArgs, ChildActivitySelectedArgs, ContainerSelectedArgs, EditChildActivityArgs, GraphUpdatedArgs, WorkflowUpdatedArgs } from '../../models';
+import { createGraph } from './graph-factory';
+import { AddActivityArgs, Connection, Flowchart, FlowchartModel, FlowchartPathItem, LayoutDirection, RenameActivityArgs, UpdateActivityArgs } from './models';
+import { NodeFactory } from "./node-factory";
+import { Container } from "typedi";
+import { createActivityLookup, EventBus, flatten, PortProviderRegistry, walkActivities } from "../../services";
+import { ConnectionCreatedEventArgs, FlowchartEvents } from "./events";
+import { ContextMenuAnchorPoint, ContextMenuItem } from "../../components/shared/context-menu/models";
```

Comment:
Many changes in this PR relate to different code style settings between your and my IDE. Although I like your styling better, it would be best if you can adjust your settings (ideally, via some config file part of this repo) and update your PR to only include the relevant changes to support readonly workflows.